### ISSUE TITLE
Unable to input Japanese text

### ISSUE THREAD

**Issue by tatsu38** [extraymond/zotero-snap#22]
I use Zotero conveniently. Recently, I migrated to the Snap package. However, I cannot input Japanese text.

I am not familiar with it, but adding desktop interfaces according to the following documentation might solve it. I hope you can fix it.

https://snapcraft.io/docs/desktop-interfaces

**Comment by extraymond** [comment]
Thanks, I can take a look at it, may I ask what's input method's framework you are using?

**Comment by tatsu38** [comment]
The input method I am using is ibus-mozc. This is the default for the Japanese version of Ubuntu 22.04 (not the flavor).

**Comment by extraymond** [comment]
Can you try the beta branch and see if that works for you?

**Comment by tatsu38** [comment]
Thank you for adding desktop-legacy and desktop to the beta version.

Before this update, I had experimented locally. After doing so, I was able to input Japanese without any problems without changing anything in the source code.

So I reinstalled the stable version of zotero-snap and still no problem. Maybe it has something to do with the fact that I upgraded from Ubuntu 20.04 to 22.04 a few days ago.

I will try the beta version now.
